website/static/js/profile.js

```diff
@@ -577,9 +613,45 @@ var ListViewModel = function(ContentModel, urls, modes) {
         }
         return true;
     });
+
+    self.institutionsEmpty = ko.computed(function() {
```

Comment:
@sloria, you had some questions about "institution" being in the ListView.  "institution" is the generic name for the title field of both schools and jobs in our profile.  It doesn't seem like we need to refactor unless we are changing the structure of the objects.  Let me know if you think differently.